core/controllers/editor.py

```diff
@@ -71,6 +71,34 @@ def _require_valid_version(version_from_payload, exploration_version):
 }
 
 
+def get_exploration_rights_with_names(exploration_id):
+    """Replace ids with the names and return rights as a dictionary.
+
+    Args:
+        exploration_id: str. ID of the exploration.
+
+    Returns:
+        dict. The dictionary where ids are replaced with corresponding names.
+    """
+
+    rights_dict = rights_manager.get_exploration_rights(
+        exploration_id).to_dict()
+    ids = ['owner_ids', 'editor_ids', 'voice_artist_ids', 'viewer_ids']
```

Comment:
Please try to use more descriptive variable name.

Reply:
Done.